src/sagemaker_training/smdataparallel.py

```diff
@@ -35,19 +35,24 @@
 logger = logging_config.get_logger()
 logging.getLogger("paramiko").setLevel(logging.INFO)
 
-try:
-    from smdistributed.dataparallel import exceptions
-
-    # list of exceptions SMDDP wants training toolkit to catch and log
-    exception_classes = [x for x in dir(exceptions) if isclass(getattr(exceptions, x))]
-# relaxed exception type in case of custom exceptions thrown during import
-except Exception:  # pylint: disable=broad-except
-    logger.info(
-        "smdistributed.dataparallel not found or "
-        "using an older version without custom exceptions."
-        "SM training toolkit will track user script error only"
-    )
-    exception_classes = [errors.ExecuteUserScriptError]
+
+def get_dataparallel_exception_classes():
+    """Get ddp exception classes"""
+    try:
```

Comment:
we should define  exception_classes to be an empty list first. Just to follow the convention of the other two methods.